packages/protocol/contracts/common/FeeHandler.sol

```diff
@@ -384,6 +397,10 @@ contract FeeHandler is
     return tokenStates[tokenAddress].toDistribute;
   }
 
+  function getTokenToBurn(address tokenAddress) external view returns (uint256) {
+    return tokenStates[tokenAddress].toDistribute;
```

Comment:
Should this be rather toBurn (not toDistribute)

Reply:
very true